lib/custom/src/MShop/Service/Provider/Payment/OmniPay.php

```diff
@@ -700,6 +721,11 @@ protected function getData( \Aimeos\MShop\Order\Item\Base\Iface $base, $orderid,
 			$data['card'] = $this->getCardDetails( $base, $params );
 		}
 
+
+		if( $this->getValue( 'token', false ) && isset($params['paymenttoken']) ){
```

Comment:
This seems to be Stripe specific too

Reply:
I thought if we will have more payments systems with tokens, we can keep this code here, so it's checking if child class have tokens. Okay I will move it to child Stripe class